packager/mpd/base/xml/xml_node_unittest.cc

```diff
@@ -6,20 +6,24 @@
 
 #include <gflags/gflags.h>
 #include <gmock/gmock.h>
+#include <google/protobuf/text_format.h>
```

Comment:
By using `ConvertToMediaInfo` from `mpd_builder_test_helper.h`, do we still need the includes in line 9 and line 17?